tendrl/monitoring_integration/flows/new_cluster_dashboard/__init__.py

```diff
@@ -72,6 +73,10 @@ def create_all_dashboard(self, dashboard_name, cluster_detail_list):
                            "with error {0}".format(str(error))})
 
     def create_resource(self, integration_id, cluster_detail_list, resource_type):
+        if reource_type == "volumes":
```

Comment:
There's typo "reource_type" , is this even verified?

Reply:
@rishubhjain please make sure this function is calling